collect-adb.rst

```diff
@@ -0,0 +1,115 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+This document focuses specifically on using **adb** as a command line tool to perform tasks related to Collect. 
+
+.. _what-is-adb:
+
+What is Android Debug Bridge (adb)?
+====================================
+
+Android Debug Bridge or `adb <https://developer.android.com/studio/command-line/adb.html>`_ is a command line tool which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands, and more. To install **adb**, please follow the instructions given `here <https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/>`_. For the developers and users of ODK Collect, the most common uses are:
```

Comment:
Link "Android Debug Bridge", delete `adb` in first sentence. Formally, `adb` is the command, not the tool.

 A lot of extra words here.

> Android Debug Bridge is a command line tool which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands, and more. (43 words)

Possible correction:

> Android Debug Bridge is a tool that gives you command line access to an Android device. (16 words)

Don't mention installing here. Say what it is, and then why you need it, AND THEN how to get it.

> Android Debug Bridge is a tool that gives you command line access to an Android device. For ODK Collect users and developers, the most common uses are: